### ISSUE TITLE
Request - Soul Eater Rebalance

### ISSUE THREAD

**Issue by Utildayael** [TeamFRM/TAIGA#129]
Been enjoying TAIGA but I've noticed one major glaring issue. The Soul Eater material modifier is insanely OP. I make a ninja star with Nihilite and got about a thousand kills on it. It's like +20 damage already! 

A friend took his Nihilite rapier and tossed it into an ExtraUtils2 Mechanical User so it would auto-use it on a mob farm. He came back later to about 6000 registered kills on t and a modifier of like +550 damage. 

While I like the idea of it growing power, it grows too fast and needs a max cap or something. Right now it's the go-to method for making insane weapons. 

Add to this using tinkers leveling and/or paper you can literally make the weapon unbreakable (5x reinforced traits). 

So... in short. 

- Reduce bonus on Soul Eater by exponential values or change the enchant completely. Maybe make it a temporary buff or something?

- Reduce the rate at which Soul Eater scales per kill

- If possible, make Soul Eater aware of its in a machine vs the player hand. I'm not sure if this can be done so to how the actuators / mech user / etc work as they act like a ghost player. Maybe could bind the tool to an "owner" and check player name vs tool owner? A few mods do this with machines to limit who can muck with settings or use them so just a thought. Pain in the ass either way for you to deal with. :)

Personally I would just redo the trait to just apply darkness, wither or something to the mob. 

Cheers

**Comment by Cheezy16** [comment]
I have one question: How? Vibranium still cant be melted, unless im really missing something. I havent really used dilithium yet but JEI believes that no fuels will melt it.

**Comment by Utildayael** [comment]
You can easily melt vibranium just by with lava. Magma fluid for one will do it. So will the IE Arc Furnace.

**Comment by Cheezy16** [comment]
I updated today... Magma fluid is still only obtained through using liquids
which require already having it. Vibranium for sure can't be melted in
lava. I'm trying to make parts, and I already crushed the ore.

**Comment by Utildayael** [comment]
I believe I had to make a diff fluid first then use that to make magma fluid. But yes it can be done because I literally just did it a few weeks ago. :)

Deuterium fluid maybe? Can't remember the names. TAIGA isn't the easiest to follow from a progression standpoint. There are a few videos on YouTube that go over all the stuff. I think the one I watched was by TheyCallMeGunny or something like that. He covets everything.

**Comment by Cheezy16** [comment]
Hmm.. OK, I'll check tomorrow then. Thanks a lot.

**Comment by DRKILLY** [comment]
I agree, Soul Eater should have a max cap somewhere. Infinite damage through mob killing is gamebreaking.
